Formula/php.rb

```diff
@@ -81,6 +83,10 @@ def install
     # Prevent homebrew from harcoding path to sed shim in phpize script
     ENV["lt_cv_path_SED"] = "sed"
 
+    # system pkg-config missing
+    ENV["KERBEROS_CFLAGS"] = " "
+    ENV["KERBEROS_LIBS"] = "-lkrb5"
```

Comment:
```suggestion
    ENV["KERBEROS_LIBS"] = "-lkrb5"
    ENV["SASL_CFLAGS"] = "-I#{MacOS.sdk_path_if_needed}/usr/include/sasl"
    ENV["SASL_LIBS"] = "-lsasl2"
    ENV["EDIT_CFLAGS"] = " "
    ENV["EDIT_LIBS"] = "-ledit"
```

I needed some more `ENV` variables to get this formula to work.